templates/etc/aptly.init.erb

```diff
@@ -0,0 +1,12 @@
+description "Aptly API Service"
+author "wilrnh"
```

Comment:
The `author` stanza is optional. Are you attached to having it here? It would make this much easier to merge if it wasn't, else we'd need to speak to lawyers for $reasons.